dubbo-configcenter/dubbo-configcenter-zookeeper/src/main/java/org/apache/dubbo/configcenter/support/zookeeper/CacheListener.java

```diff
@@ -35,6 +35,8 @@
  */
 
 public class CacheListener implements DataListener {
+    private static final int MIN_PATH_ARRAY_LENGTH = 5;
```

Comment:
how about name it as MIN_PATH_DEPTH

Reply:
"MIN_PATH_DEPTH" is better. Thank you for your advice. I will rename the constant.